appveyor.yml

```diff
@@ -0,0 +1,41 @@
+environment:
```

Comment:
you have a rebasing issue again @monkeyman192 

I can't merge until it's fixed ...

Reply:
I think I fixed the rebase issue at least partially. But now we have an issue due to the final commit which removed the doubled up os check in test_meg_bids.py.
If you don't care about history I can make a new branch which has all the changes applied correctly then just replace the master branch with that one then all the fixes should be correct. But it will all be condensed into a single commit.

Reply:
It's not a lot of changes. I'm okay with condensing it all into a single commit :)